scripts/skills/perks/perk_rf_tempo.nut

```diff
@@ -107,6 +109,8 @@ this.perk_rf_tempo <- ::inherit("scripts/skills/skill", {
 	function onTargetMissed( _skill, _targetEntity )
 	{
 		this.gainStackIfApplicable(_skill, _targetEntity);
+		if (this.m.AttacksRemaining > 0 && ::Tactical.TurnSequenceBar.isActiveEntity(this.getContainer().getActor()))
```

Comment:
You do not check, whether _skill is an attack. But during the hit-check you check for that. That seems like an oversight, even if there are currently none/few? skills which are not an attack but can miss.

Reply:
The `isAttack` check is redundant as both `onBeforeTargetHit` and `onTargetMissed` are called from `onScheduledTargetHit` and `attackEntity` functions of skill.nut which are for attacks only. So I will remove the check.